api.md

```diff
@@ -1990,6 +1990,19 @@ filling for the interval between `start` and `finish`. It can only be used with
 query. Values outside of `start` and `finish` will pass through but no gap filling will be
 done outside of the specified range.
 
+Starting with version 1.3.0, `start` and `finish` are optional arguments and will
+be inferred from the WHERE clause if not supplied as arguments.
+
+>:TIP: We recommend using a WHERE clause whenever possible (instead of just
+ `start` and `finish` arguments), as TimescaleDB will then properly perform
+ chunk exclusion having been supplied the WHERE predicate.  Chunk exclusion
+ will not occur if only `start` and `finish` arguments are supplied.
+
+The `time_bucket_gapfill` must be a top-level expression in a query or
+subquery, as shown in the above examples.  You cannot, for example, do
+something like `round(time_bucket_gapfill(...))` or cast the result the gapfill
```

Comment:
```suggestion
something like `round(time_bucket_gapfill(...))` or cast the result of the gapfill
```

Reply:
since chunk exclusion requires a filter this seems to suggest that start and stop will be applied as filter which is not the case, start and stop only controls the interval where gapfilled tuples will be injected